### PR TITLE
review

### PR DESCRIPTION
**Pull request by rkolpakov**

### CODE REVIEW

File: src/styles/simple-reset.css [hunk 1]
```diff
@@ -1,44 +0,0 @@
-html, body, div, span, applet, object, iframe,
```
Comment (evgenyrodionov) [inline src/styles/simple-reset.css]: Почему не normalize.css?

File: src/store.js [hunk 2]
```diff
@@ -1,19 +0,0 @@
-import { createStore, applyMiddleware } from 'redux'
```
Comment (evgenyrodionov) [inline src/store.js]: В чём смысл делать функцию, а потом вызывать её в app.js? Почему бы не экспортить готовый store отсюда?

File: src/reducers/page.js [hunk 3]
```diff
@@ -1,27 +0,0 @@
-const initialState = {
```
Comment (evgenyrodionov) [inline src/reducers/page.js]: Почему редьюсер называется page?

File: src/containers/pagination/index.js [hunk 4]
```diff
@@ -1,47 +0,0 @@
...
-      <div className='pagination'>
-        <UI.Button onClick={this.prev.bind(this)}>Prev</UI.Button>
-        <UI.Button onClick={this.next.bind(this)}>Next</UI.Button>
-      </div>
-    )
-  }
-}
-
-function mapStateToProps(state) {
-  return {
```
Comment (evgenyrodionov) [inline src/containers/pagination/index.js]: Почему не `return state.page`?

File: src/containers/pagination/index.js [hunk 5]
```diff
@@ -1,47 +0,0 @@
...
-class Pagination extends React.Component {
-  constructor(props) {
-    super(props)
-  }
-  next() {
-    if (this.props.offset + this.props.limit <= this.props.totalCount) {
-      this.props.pageActions.loadObjects(this.props.offset + this.props.limit)
-    }
-  }
-  prev() {
```
Comment (evgenyrodionov) [inline src/containers/pagination/index.js]: Лучше сделать одной функцией handleChange, в которую ты передаёшь направление.
И кнопки лучше отключать, а не тихо не пропускать экшен — пользователь не поймёт, почему кнопка не сработала

File: src/containers/pagination/index.js [hunk 6]
```diff
@@ -1,47 +0,0 @@
-import React from 'react'
```
Comment (evgenyrodionov) [inline src/containers/pagination/index.js]: Пагинатор можно сделать компонентом, в который ты из `containers/app` спускаешь экшены

File: src/containers/app/index.js [hunk 7]
```diff
@@ -1,45 +0,0 @@
...
-  constructor(props) {
-    super(props)
-  }
-  componentWillMount() {
-    this.props.pageActions.loadObjects()
-  }
-  render() {
-    return (
-      <div className='app'>
-        <Preloader show={this.props.page.preloader} />
```
Comment (evgenyrodionov) [inline src/containers/app/index.js]: show → isShow, иначе непонятно, что ты спускаешь булеву переменную

File: src/components/preloader/index.js [hunk 8]
```diff
@@ -1,7 +0,0 @@
-import React from 'react'
-import './preloader.css'
-
-export default (props) => {
-  let preloader = <div className='layer'><div className='spin'></div></div>
```
Comment (evgenyrodionov) [inline src/components/preloader/index.js]: Почему не const? И почему отдельной переменной, в чём смысл? 

```
if (props.isShow) {
  return <div />
} else {
  return null
}
```

File: src/components/card/index.js [hunk 9]
```diff
@@ -1,27 +0,0 @@
...
-          <div className='bottom__price'>{getPrice(props.item.saleOffer || props.item.rentOffer)}</div>
-          <div className='bottom__area bottom-separator'>{props.item.landDetails.area} сот</div>
-          <div className='bottom__house-area bottom-separator'>{props.item.specification.area} м<sup>2</sup></div>
-        </div>
-      </div>
-    </div>
-  )
-}
-
-const getPrice = (data) => {
```
Comment (evgenyrodionov) [inline src/components/card/index.js]: https://github.com/jqestate/react-formatted/blob/master/src/currency.jsx

File: src/actions/pageActions.js [hunk 10]
```diff
@@ -1,31 +0,0 @@
...
-  return (dispatch) => {
-    dispatch({
-      type: 'LOADING_OBJECTS',
-      preloader: true
-    })
-
-    let url = 'https://api.jqestate.ru/v1/properties/country'
-
-    if (offset) {
-      url += '?pagination[offset]=' + offset
```
Comment (evgenyrodionov) [inline src/actions/pageActions.js]: Мутабельность это плохо, плюс, я думаю, superagent поддерживает query-параметры как объект

File: src/actions/pageActions.js [hunk 11]
```diff
@@ -1,31 +0,0 @@
...
-    }
-
-    ajax.get(url)
-      .end((error, response) => {
-        if (!error && response) {
-          dispatch({
-            type: 'LOADED_OBJECTS',
```
Comment (evgenyrodionov) [inline src/actions/pageActions.js]: Если захочешь поменять название экшена, то будешь по всем файлам бегать? Такие вещи лучше выносить в константы и импортить переменными
